package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@blackbaud/skyux-lib-restricted-view",
+  "version": "1.0.0-rc.1",
+  "description": "Single-page-application for skyux-lib-restricted-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "scripts": {
+    "skyux": "skyux",
+    "test": "skyux test --coverage library",
+    "start": "skyux serve --hmr",
+    "build": "skyux build-public-library"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/common": ">=4.3.6",
+    "@angular/core": ">=4.3.6",
+    "@blackbaud/auth-client": "^2.0.0",
+    "@skyux/http": "^3.0.0"
+  },
+  "dependencies": {
+    "jwt-decode": "2.2.0"
```

Comment:
Due to the recent change in `@skyux/http`, we can remove this dependency (see comment below).